dir.targets

```diff
@@ -69,5 +69,20 @@
   <Target Name="BuildAndTest" DependsOnTargets="Build;Test" />
 
   <Target Name="Test" />
+  
```

Comment:
No I mean actually adding the checks in the target like Build.Common.targets or somewhere in those targets.